templates/module/src/Controller/controller.php.twig

```diff
@@ -22,7 +22,7 @@ use Symfony\Component\DependencyInjection\ContainerInterface;
  *
  * @package Drupal\{{ module }}\Controller
  */
-class {{ class_name }} extends ControllerBase {% if services is not empty %}implements ContainerInjectionInterface {% endif %}
+class {{ class_name }} extends ControllerBase
```

Comment:
@danielrose28: Remove new line at end of line 25, make sure you let and empty space between `ControllerBase` and `{% endblock %}` it should look like this:
```
class {{ class_name }} extends ControllerBase {% endblock %}
```